core/src/main/java/nu/yona/server/entities/EntityWithUuid.java

```diff
@@ -9,7 +9,9 @@
 import javax.persistence.Id;
 import javax.persistence.MappedSuperclass;
 
+import nu.yona.server.goals.entities.Goal;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'nu.yona.server.goals.entities.Goal'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AUselessImportCheck)